lib/AtteanX/Serializer/SPARQLHTML.pm

```diff
@@ -141,6 +141,22 @@ Serializes the L<Attean::API::Term> object as HTML.
 				s/</&lt;/g;
 			}
 			my $html	= $uri;
+
+			if (my $map = $self->namespaces) {
+				my $abr = $map->abbreviate($html);
+
+				if ($abr) {
+					return qq[<a href="${html}">$abr</a>];
+				} else {
+					return $html;
+				}
+
+			} else {
+			
+				return $html;
```

Comment:
No problem! Looks like I will have to modify the t/serializer-sparqlhtml.t too since some tests expect the plain IRIs. Or perhaps you want this change only when there is URI::NamespaceMap instance?